packages/util/lib/src/json_converter.dart/color_converter.dart

```diff
@@ -11,6 +11,7 @@ class ColorConverter implements JsonConverter<Color, int> {
   const ColorConverter();
 
   @override
+  // ignore: avoid_hardcoded_color
```

Comment:
In order to get analyzer/test to run, I enabled custom linting in the util package to handle this.
This is a JsonConverter and needs to use `Color()`, so it was IGNORED.